File: 체육복/javascript/20221103/solution.test.js
import solution from './solution';

test('체육 수업을 들을 수 있는 최종 인원을 구해라.', () => {
  expect(solution(5, [2, 4], [1, 3, 5])).toBe(5);
  expect(solution(5, [2, 4], [3])).toBe(4);
  expect(solution(3, [3], [1])).toBe(2);
});

File: programmers/level1/비밀지도/java/20221007/test/SolutionTest.java
import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.*;

class SolutionTest {

    @Test
    void convertToBinary() {
        Solution solution = new Solution();

        assertArrayEquals(new String[]{"11111", "10101", "11101", "10011", "11111"},
                solution.convertToBinary(5, new int[]{9, 20, 28, 18, 11}, new int[]{30, 1, 21, 17, 28}));
    }

    @Test
    void decodeSecretMap() {
        Solution solution = new Solution();

        assertArrayEquals(new String[]{"#####","# # #", "### #", "#  ##", "#####"},
                solution.decodeSecretMap(5, new String[]{"11111", "10101", "11101", "10011", "11111"}));
    }
}
